src/style/table.rs

```diff
@@ -46,7 +46,7 @@ pub enum ContentArrangement {
 /// |   |   |   |    The inner "+" chars are MiddleIntersections
 /// +---+---+---+
 /// ```
-#[derive(Debug, PartialEq, Eq, Hash, EnumIter)]
+#[derive(Debug, PartialEq, Eq, Hash, EnumIter, Clone, Copy)]
```

Comment:
Why do you want to declare this as copy? Working with references should work in (almost?) all cases and automatic copy might be unwanted behavior in some circumstances.

Reply:
the reason i've done this is a little subtle- since this is a unit enum, it's actually slightly more efficient to make it `Copy` and pass it by value, than to have it be non-`Copy` and pass it by reference